source/install/config-proxy-nginx.rst

```diff
@@ -0,0 +1,88 @@
+.. _config-proxy-nginx:
+
+Configuring NGINX as a proxy for Mattermost Server
+==================================================
+
+**To configure NGINX as a proxy**
+
+1. Log into the server that hosts NGINX and open a terminal window.
+2. Create a configuration file for Mattermost.
```

Comment:
@JeffSchering Ah good point about scanning through the commands, it does make it easier for someone to just run through them on the terminal.

Let's keep it the way it is now, thanks @JeffSchering!